index.js

```diff
@@ -639,7 +639,16 @@ function launchApp(argo, startExternalAdapterServer) {
         if (uuid && !isRunning) {
             if (!lockUuid(uuid)) {
                 deleteApp(uuid);
-                duplicateUuidTransport.broadcast({ argv, uuid });
+                // We need to rebuild a new argv to have correct app info in it.
+                let newArgv = Object.keys(argo).map(key => {
+                    if (key === '_') {
```

Comment:
would this value ever be an array with a length other than 1? I believe it's all arguments that can't be split into flag/value pairs

Reply:
The value is array, but not name-value pair in array.